src/database/bank_database.rb

```diff
@@ -0,0 +1,46 @@
+# a class created to simulate a bank database with user accounts, bank accounts and an ATM unit
+class BankDatabase
+
+    # constructor
```

Comment:
I wouldn't say this kind of comments are useful, it's quite obvious that this method is a constructor. In general, if I feel the need to write a comment I'm thinking why I need that comment and try to make the code cleaner, so I don't need it at all :D